src/cmd_line/subparser.ts

```diff
@@ -348,6 +349,11 @@ export const commandParsers = {
     parser: fileCmd.parseEditNewFileInNewVerticalWindowCommandArgs,
   },
 
+  vsc: {
```

Comment:
I think a full name of `vscode` makes sense

Reply:
@J-Fields Updated. Thanks!